.vib/nats/goss/nats.yaml

```diff
@@ -0,0 +1,19 @@
+command:
+  check-nats-server:
+    exec: timeout 5 nats-server || true
+    timeout: 10000
+    exit-status: 0
```

Comment:
Going by what we decided regarding the use of `|| true`, could you please update this test?